package.json

```diff
@@ -33,12 +34,22 @@
             "make_targets": {
                 "darwin": [
                     "zip"
+                ],
+                "win32": [
+                    "squirrel"
                 ]
             },
             "electronPackagerConfig": {
                 "packageManager": "npm",
                 "icon": "assets/icon.icns"
             },
+            "electronWinstallerConfig": {
```

Comment:
Also need to convert icon to windows format.
You can use [icoutils](http://www.nongnu.org/icoutils/) for that:
`icontool -c icon.png > icon.ico`